main/src/com/google/refine/operations/cell/FillDownOperation.java

```diff
@@ -63,15 +64,17 @@ public FillDownOperation(
 
     @Override
     protected String getBriefDescription(Project project) {
-        return "Fill down cells in column " + _columnName;
+        // return "Fill down cells in column " + _columnName;
+        return OperationDescription.cell_fill_down_brief(_columnName);
     }
 
     @Override
     protected String createDescription(Column column,
             List<CellChange> cellChanges) {
         
-        return "Fill down " + cellChanges.size() + 
-            " cells in column " + column.getName();
+        // return "Fill down " + cellChanges.size() +
+        //    " cells in column " + column.getName();
```

Comment:
I would say it is worth removing those comments before merging.

Reply:
On it ✌🏾